index.html

```diff
@@ -370,11 +370,11 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
       <p data-lang="zh-hans">汉字依行排列文字，原则上文字外框彼此紧贴配置，称作密排。</p>
       <p data-lang="zh-hant">漢字依行排列文字，原則上文字外框彼此緊貼配置，稱作密排。</p>
       <div class="note">
-        <p data-lang="en" class="checkme">Unlike the letterpress printing era, when several sizes of the original pattern of a letter were required to create matrices, in today's digital era the same original pattern is used for any size simply by enlargement or reduction. Because of this, it might be necessary to adjust the inter-character space when composing lines at large character sizes. When composing lines at small character sizes in outline fonts, hinting data is used to ensure that the width of the strokes that make up a character look correct. When only a small portion of the fonts are smaller, they will be displayed in bitmaps, and there is no need to make extra adjustments.</p>
+        <p data-lang="en" class="checkme">Unlike the letterpress printing era, when several sizes of the original pattern of a letter were required to create matrices, in today's digital era, the same original pattern can be used for any size simply by enlargement or reduction. Because of this, it might be necessary to adjust inter-character spacing when composing lines at large character sizes. When composing lines at small character sizes in outline fonts, hinting data is used to ensure that the width of the strokes that make up a character look correct. When only a small portion of the fonts are smaller, they will be displayed in bitmaps, and there is no need to make extra adjustments.</p>
```

Comment:
Not too sure about this one. Re-read the Chinese version again and came up with this:
"From the advent of moveable type, each character was set to be flush with one another without any gaps, regardless of vertical or horizontal writing mode. However back then, several sizes of the original pattern of a letter were required to create matrices,"